kornia/augmentation/augmentation.py

```diff
@@ -2168,6 +2170,7 @@ def __init__(
         alpha: Tuple[float, float] = (1.0, 1.0),
         align_corners: bool = False,
         mode: str = 'bilinear',
+        padding_mode: Union[str, int, SamplePadding] = SamplePadding.ZEROS.name,
```

Comment:
Types should go according to th fucntional, this should be string

Reply:
fixed